wandb/filesync/upload_job.py

```diff
@@ -14,7 +15,7 @@
 
 class EventJobDone(NamedTuple):
     job: "UploadJob"
-    success: bool
+    exception: Optional[Exception]
```

Comment:
For some reason I find myself shying away from storing an exception as normal data, but I see the reason here. (no change requested)

Reply:
Yeaaah, this made me go 🤔 too, but I couldn't think of a better alternative. Thanks for the vindication!